third_party/argon2/BUILD.gn

```diff
@@ -0,0 +1,34 @@
+config("warnings") {
+  cflags = []
+
+  if (is_clang) {
+    cflags += [ "-Wno-implicit-fallthrough" ]
+  }
+}
+
+config("external_config") {
+  include_dirs = [ "//brave/third_party/argon2/src/include" ]
+}
+
+static_library("argon2") {
+  configs += [ ":warnings" ]
+  public_configs = [ ":external_config" ]
+
+  visibility = [ "//brave/browser/brave_wallet" ]
+
+  sources = [ "src/src/argon2.c" ]
+
+  sources += [
+    "src/src/blake2/blake2-impl.h",
+    "src/src/blake2/blake2.h",
+    "src/src/blake2/blake2b.c",
+    "src/src/blake2/blamka-round-ref.h",
+    "src/src/core.c",
+    "src/src/core.h",
+    "src/src/encoding.c",
+    "src/src/encoding.h",
+    "src/src/ref.c",
+    "src/src/thread.c",
```

Comment:
we should define `ARGON2_NO_THREADS` and not include these

Reply:
fixed in c87071ca99ba1cf35fb6795d402c6d5c6c4394fb